fetch-npm-react-native.js

```diff
@@ -0,0 +1,3 @@
+// Self is defined in Chrome Debugger, but not in JavaScriptCore
+var globalObject = typeof self === "undefined" ? global : self;
+module.exports = globalObject.fetch.bind(globalObject);
```

Comment:
@vjeux Because I didn't think of this :) I just adapted the solution in #63 without too much thought…